docs/docs/01-start-here/04-run-locally.md

```diff
@@ -49,6 +49,10 @@ You can also download the file, using the **Download** button.
 
 The downloaded file should contain `Hello, Wing it` text (as you already saw in the preview).
 
+You can push any number of messages to the queue and watch it fill up. Each time a message is pushed to the queue, you can see the counter's value is incremented via the `Actual value` field in the counter resource. It is even possible to reset the counter value with the `Reset value` button.
```

Comment:
hmm, just thought about this phrasing, maybe we should remove the `watch it fill up` bit as it could be confusing to some users that might be expecting messages to accumulate but are not because the function processes them as they are pushed.

```suggestion
You can push any number of messages to the queue. Each time a message is pushed to the queue, you can see the counter's value is incremented via the `Actual value` field in the counter resource. It is even possible to reset the counter value with the `Reset value` button.
```